### DISCUSSION TITLE
How to use FindMarkers() to find the intergenic da peaks?

### DISCUSSION THREAD

**Discussion by jiangpuxuan** [stuart-lab/signac#1046] [Q&A] [answered]
I think this problem might sounds silly, but I was really confused about it. 
I have such kind of intergenic da peaks in my dataset. Obviously, it is cell-type specific.
But I cannot find it by `FindMarkers()`. What parameters should I use?
Thank you for your help!
<img width="878" alt="008551c025eb2c04d041b3dbb811f4e" src="https://user-images.githubusercontent.com/81287604/158811645-c2c69fe9-843f-4288-b895-709f662bf4f9.png">

**Comment by timoast** [comment]
You need to ensure that these peaks are identified in your peak calling, and quantified in the matrix used to create the assay. You can view the peak regions by setting `peaks=TRUE` in `CoveragePlot()` (which is set by default), to assess the accuracy of peak calling. If a peak is not identified you can't run a DA test on it. I would suggest running `CallPeaks()` and optimizing the MACS2 parameters until you obtain a set of peaks that you're happy with.

**Comment by jiangpuxuan** [comment, marked as answer]
I made it by `COSG`. `COSG` have some quite advantages but a lot of weird features........
